pymesos/subprocess/operator_v1.py

```diff
@@ -0,0 +1,30 @@
+from .. import OperatorMaster
```

Comment:
Why we need this?

Reply:
We don't actually: the ProcOperatorMaster class is not used anywhere.
The passed OperatorMaster in the MesosOperatorMasterDriver constructor is optional: if no OperatorMaster is passed, then the event methods are not intended to be called.

But we may think of a different behavior: if no OperatorMaster is passed, then the default ProcOperatorMaster is used which just logs the Mesos callbacks.

Make your choice, and I'll do the necessary changes if any.

Reply:
Hmm... I think we do not need this for now.  `pymesos.subprocess` is an in-place replacement for `subprocess` on Mesos, thus we do not need an `Operator` implementation.